site/docs/tasks/data-plane-optimization.md

```diff
@@ -0,0 +1,19 @@
+# Enabling data-plane optimization
+
+Fybrik takes into account data governance and hard IT config policies when building a data plane. However, it does not by default take into account **soft** IT config policies (i.e., optimization goals). To enable data-plane optimization, the [Optimizer component](../concepts/optimizer.md) must be enabled.
```

Comment:
IT config optimization policies ... instead of soft

Reply:
done